src/minimap_utils_bonus.c

```diff
@@ -0,0 +1,242 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   minimap_utils_bonus.c                              :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: araveala <araveala@student.hive.fi>        +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/10/09 08:37:11 by araveala          #+#    #+#             */
+/*   Updated: 2024/10/17 12:39:09 by araveala         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "../inc/cubd.h"
+
+/**
+ * Takes player position and adjusts it based on minimapp size, minimap
+ * consists of 6 x 6 tiles. This assumes we want player in center of minimap.
+ * @param p_x x_ppos type cast to int
+ * @param p_y y_ppos type cast to int
+ * int type casting may be oboslete depending on decision of minimap handeling
+ */
+void	adjust_mapstart(int *p_x, int *p_y)
```

Comment:
I did not quite understand what this function is doing at first. It'd help a bit to be more verbose on the variable names.